packages/utils/src/bulk-query-patients.ts

```diff
@@ -106,26 +106,34 @@ async function queryDocsForPatient(
       resources: {},
       total: 0,
     };
+    const getPatientPromise = async () => metriportAPI.getPatient(patientId);
+    const patient = await getPatientPromise();
```

Comment:
No need to declare the "get promise" function if we're just calling it here - might as well just have

```ts
const patient = await metriportAPI.getPatient(patientId);
```